modules/updating-eus-to-eus-layered-products.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-olm-operators_{context}"]
+= EUS-to-EUS update for layered products and Operators installed through Operator Lifecycle Manager (OLM) 
+
+In addition to the EUS-to-EUS update steps mentioned for the web console and CLI, there are a couple of additional steps to consider when performing EUS-to-EUS updates for the clusters of layered products and those of Operators that have been installed through the Operator Lifecycle Manager (OLM). 
+
+.What is a layered product?
+
+Layered products refer to products that are made of multiple underlying products that are intended to be used together and cannot be broken into individual subscriptions. For examples of layered {product-title} products, see link:https://access.redhat.com/support/policy/updates/openshift/#layered[Layered Offering On OpenShift].
+
+As you perform an EUS-to-EUS update for the clusters of layered products and those of Operators that have been installed through OLM, you must complete the following:
+
+. Ensure that all of your Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures that they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update. For information on how to update your Operators, see _Preparing for an Operator update_ in Additional resources.
+
+. Confirm the API and kernel compatibility used in the current and intended Operator versions. You can verify which versions your OLM operators are compatible with by using the link:https://access.redhat.com/labs/ocpouic/?operator=logging&&ocp_versions=4.10,4.11,4.12[Red Hat {product-title} Operator Update Information Checker].
+
+As an example, here are the steps to perform an EUS-to-EUS update from <4.y> to <4.y+2> for OpenShift Data Foundation (ODF). This can be done through the CLI or web console. For information on how to update clusters through your desired interface, see _EUS-to-EUS update using the web console_ and _EUS-to-EUS update using the CLI_ in Additional resources.
+
+.Procedure:
+. Pause the worker machine pools
+. Upgrade OpenShift <4.y> → OpenShift <4.y+1>
+. Upgrade ODF <4.y> → ODF <4.y+1>
+. Upgrade OpenShift <4.y+1> → OpenShift <4.y+2>
+. Upgrade to ODF <4.y+2>
+. Unpause the worker machine pools
+
+Upgrade to ODF <4.y+2> can happen before or after worker machine pools have been unpaused.
+
+In the ODF example above, OpenShift Container Storage (OCS) <4.y> is compatible with OpenShift <4.y> and OpenShift <4.y+1>, and ODF <4.y+1> is compatible with OpenShift <4.y+1> and OpenShift <4.y+2>.
```

Comment:
In general (and it's very likely I am just not familiar with some aspect of the subject matter at play here), I wasn't really sure what this last bit was trying to illustrate, as far as bringing up OCS.

Reply:
checked w/ engineering, and it looks like it's just something that was top of mind that a customer had bumped up against. No need to call OCS out specifically, so will remove :)